core/core-toggle.el

```diff
@@ -95,10 +95,12 @@ used."
                                    ,condition)
                            t)))
     `(progn
-       (push (append '(,name)
-                     '(:function ,wrapper-func :predicate ,wrapper-func-status)
-                     ',props)
-             spacemacs-toggles)
+       (let ((tail (append '(:function ,wrapper-func :predicate ,wrapper-func-status)
+                           ',props))
+             (cell (assq ',name spacemacs-toggles)))
+         (if cell
+             (setcdr cell tail)
+           (push (cons ',name tail) spacemacs-toggles)))
```

Comment:
Right, I am following the example of `spacemacs|diminish`.  (The same idiom appears in a few places in Spacemacs; maybe it would be worth adding a macro.)  I could use `add-to-list` here instead, but as you said, that would prevent redefinition of toggles.

I'll rename `tail` to `properties`, but `cell` is a cons cell (if it's not `nil`), and we don't want to shadow the existing `name` variable, so I don't understand what you mean regarding its naming.